packages/plugins/rate-limiter/src/index.ts

```diff
@@ -24,14 +36,18 @@ export type RateLimiterPluginOptions = {
     context: unknown;
     info: GraphQLResolveInfo;
   }) => void;
+  graphQLRateLimitConfig?: GraphQLRateLimitConfig;
 };
```

Comment:
Perhaps we can just join the 2 types ? This way, all options are directly available, it's better for API discovery for users that don't have pre-existing experience of `graphql-rate-limit` :-)

We just have to make sure to omit the option that we have renamed (not sure why we did this in the first place, but not a big deal, not sure it is worth the breaking change to rename it)

```suggestion
} & Omit<GraphQLRateLimitConfig, 'identifyContext'>;
```